src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -533,6 +534,7 @@ def migrate_layers():
                 util.run_task_sync(refresh_wfs, [workspace, layername], post_task_kwargs)
             except BaseException:
                 failed_steps.append('geoserver_wfs')
+                publications.set_wfs_wms_status(workspace, LAYER_TYPE, layername, settings.EnumWfsWmsStatus.NOT_AVAILABLE)
                 logger.error(f'    Fail to recreate layer in GeoServer WFS workspace: : \n{traceback.format_exc()}')
```

Comment:
If `set_wfs_wms_status` fails for some reason, we won't see `logger.error` in logs. Therefore I prefer to call `logger.error` before `set_wfs_wms_status`.

Reply:
Incorporated.